src/splunk_otel/distro.py

```diff
@@ -41,15 +43,21 @@ def __init__(self):
         self.env = Env()
         self.logger = logging.getLogger(__name__)
 
-    def _configure(self, **kwargs):  # noqa: ARG002
+    def _configure(self, **kwargs):
         self.set_env_defaults()
+        self.set_profiling_env()
         self.configure_headers()
         self.set_server_timing_propagator()
 
     def set_env_defaults(self):
         for key, value in DEFAULTS.items():
             self.env.setdefault(key, value)
 
+    def set_profiling_env(self):
+        if self.env.is_true(SPLUNK_PROFILER_ENABLED):
```

Comment:
Explicitly state the default value of `false` here?

Reply:
Done, thx.